src/main/kotlin/model/lotto/LottoNumber.kt

```diff
@@ -0,0 +1,7 @@
+package model.lotto
+
+data class LottoNumber(val value: Int) {
```

Comment:
```suggestion
data class LottoNumber(private val value: Int) {
```

LottoNumber 를 값타입으로 사용한다면 외부에서는 value 에 접근하지 못하는게 더 좋지 않을까요?

Reply:
좋은 것 같습니다 마저 작업 좀 더 하고 올려보겠습니다